autoload/handlers.vim

```diff
@@ -265,21 +265,25 @@ def s:processHoverReply(lspserver: dict<any>, req: dict<any>, reply: dict<any>):
   endif
 
   var hoverText: list<string>
+  var hoverMarkupKind: string = ''
+  var hoverWinid: number = -1
 
   if reply.result.contents->type() == v:t_dict
     if reply.result.contents->has_key('kind')
       # MarkupContent
       if reply.result.contents.kind == 'plaintext'
-	hoverText = reply.result.contents.value->split("\n")
+        hoverText = reply.result.contents.value->split("\n")
+        hoverMarkupKind = 'plaintext'
       elseif reply.result.contents.kind == 'markdown'
-	hoverText = reply.result.contents.value->split("\n")
+        hoverText = reply.result.contents.value->split("\n")
+        hoverMarkupKind = 'markdown'
       else
-	ErrMsg('Error: Unsupported hover contents type (' .. reply.result.contents.kind .. ')')
-	return
+        ErrMsg('Error: Unsupported hover contents type (' .. reply.result.contents.kind .. ')')
+        return
```

Comment:
Note that the indentation style is to use a mix of spaces and tabs with 'shiftwidth' set to 2, 'tabstop' set to 8 and 'softtabstop' set to 2.

Reply:
yea, looks it is the same indentation style of vim source (c/c++) code. i used to set 'ts' to 4, and/or same value like sw/sts...
// i can reset this to your preferred style, or let me do it in next pr? i plan to offer a 'lsp#serverStatus' function to workaround #5.

Reply:
or actually i am a little confused, if looking into other functions or e.g 'lsp#gotoDefinition()'
```vim
  var lspserver: dict<any> = s:lspGetServer(ftype)
  if lspserver->empty()
    ErrMsg('Error: LSP server for "' .. ftype .. '" filetype is not found')
    return
  endif
```
looks many places it's using ts 2 and et yes.